### PR TITLE
Add recent API endpoints

### PR DESCRIPTION
**Pull request by Snarp**
Added support and tests for the following API endpoints:

- blog: /blocks, /followed_by, /following, /notes, /notifications, /posts/{post_id} (GET request only), /posts/queue/reorder, /posts/queue/shuffle
- user: /filtered_content

Also:

- Bumped '.travis.yml' Ruby versions to 2.6, 2.7, and 3.0.
- Bumped Gem version number to 0.8.6.

### CODE REVIEW

**Comment by fumblr** [comment]
Thanks @Snarp, this looks really useful!

Maybe someone like @cyle knows who @tumblr can review/merge/release this?

File: spec/examples/post_spec.rb [hunk 1]
```diff
@@ -20,7 +20,7 @@
       end
 
       it 'should setup a delete properly' do
-        client.delete blog_name, post_id
+        client.delete_post blog_name, post_id
```
Comment (sanmai) [inline spec/examples/post_spec.rb:23]: Looks like a BC break to me.

**Review by sanmai — APPROVED** [review]
Looks like this should do it. I'll come back in a day to merge.

**Comment by sanmai** [comment]
Thanks for all the work!

**Pull request merged**